activity/journal.go

```diff
@@ -0,0 +1,67 @@
+package activity
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+)
+
+// Journal contains the command-line used to invoke the process associated with this item
```

Comment:
Based on our conversation yesterday, I'd reword this comment to reflect that these journals are more generic than the "command-line" + "process" level.